.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: ruby
+cache: bundler
```

Comment:
I think it'd be better to leave this out - caching can be evil sometimes.

Reply:
Roger that